Formula/djl-serving.rb

```diff
@@ -46,8 +46,7 @@ def plist
     assert_predicate testpath/"#{libexec}/lib/serving-0.11.0.jar", :exist?
     shell_output("sh #{bin}/djl-serving > /dev/null 2>&1 &")
     sleep 3
-    assert_includes shell_output("curl -k -X POST 'http://127.0.0.1:8080/models?url=https%3A%2F%2Fdjl-ai.s3.amazonaws.com%2Fresources%2Ftest-models%2Fmlp.tar.gz&initial_workers=1&model_name=mlp2'"), "registered."
```

Comment:
This seems like a good test to run. Can you configure this to use a different port than `8080`? `free_port` will give you an available port. Look at other formulae that use it for examples; there are about a hundred of them.

Reply:
I actually changed the test to something simpler and which still verifies that the program ran successfully

Reply:
Yes, I can see the test you have now. What I mean is that this test is better.

Reply:
Yes, but I realized that it isn't a good test since the port is configured in the code itself. Moreover, including a link was creating a lot of problems with the test.

Reply:
CI was complaining about the line length. You could always declare the command in a separate line:
```ruby
cmd = "curl -k -X POST 'http://127.0.0.1:8080/models?url=https%3A%2F%2Fdjl-ai.s3.amazonaws.com%2Fresources%2Ftest-models%2Fmlp.tar.gz&initial_workers=1&model_name=mlp2'"
assert_includes shell_output(cmd), "registered."
```
You can even interpolate the long URL into `cmd` if `cmd` is too long.

Reply:
Ok I 've made the changes. Thank you for the help.